.pre-commit-config.yaml

```diff
@@ -34,3 +34,7 @@ repos:
         --extra-keys,
         "metadata.kernelspec metadata.vscode metadata.colab cell.metadata.executionInfo.user cell.metadata.executionInfo.user_tz cell.metadata.colab",
       ]
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  rev: v0.0.292
+  hooks:
+    - id: ruff
```

Comment:
does this run all `ruff` or just the unused imports?

Reply:
It will only run the unused imports rule ("F401"). The configuration can be found [here](https://github.com/google/flax/pull/3410/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R172-R185) and the rules list can be found [here](https://docs.astral.sh/ruff/rules/)